config/manifests/bases/falcon-helm.clusterserviceversion.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: falcon-helm.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions: {}
+  description: Helm operator for running CrowdStrike Falcon with Kubernetes
+  displayName: falcon-helm-operator
+  icon:
+  - base64data: ""
+    mediatype: ""
+  install:
+    spec:
+      deployments: null
+    strategy: ""
+  installModes:
+  - supported: false
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - security
+  - monitoring
+  - sensor
+  - EDR
+  - kubernetes
+  - alerting
+  - CrowdStrike
+  links:
+  - name: Falcon Helm
+    url: https://falcon-helm.domain
```

Comment:
Another place we might want to use the github page

Reply:
Done